local_store.go

```diff
@@ -315,11 +315,8 @@ func (f *fileSystemStore) Commit(consistentSnapshot bool, versions map[string]in
 			return err
 		}
 		if !info.IsDir() && isTarget(rel) && needsRemoval(rel) {
-			//lint:ignore SA9003 empty branch
-			if err := os.Remove(path); err != nil {
-				// TODO: log / handle error
-			}
-			// TODO: remove empty directory
+			// TODO: log / handle error or remove empty directory
+			_ = os.Remove(path)
```

Comment:
This looks dangerous. Can we restore the original code but return the err?

Reply:
In that case, should we keep or disable the linters raising these issues (for ref. when enabled this will cause the golangci-lint job to fail for this PR)?

Reply:
@rdimitrov I would fix this in a separate PR.. merge this before this one.. and then merge this one on top.

Reply:
Reverted part of the commit that introduced these changes and addressed them in https://github.com/theupdateframework/go-tuf/pull/236